config.go

```diff
@@ -54,11 +54,10 @@ type PumpConfig struct {
 	// }
 	// ```
 	Filters analytics.AnalyticsFilters `json:"filters"`
-	// You can configure a different timeout for each pump with the configuration option `timeout`.
-	// Its default value is `0` seconds, which means that the pump will wait for the writing
-	// operation forever.
+	// By default, a pump will wait forever for each write operation to complete; you can configure an optional timeout by setting the configuration option timeout.
+	// If you have deployed multiple pumps, then you can configure each timeout independently. The timeout is in seconds and defaults to 0.
 	//
-	// The format for this configuration is:
+	// The timeout is configured within the main pump config as shown here; note that this example would configure a 5 seconds timeout:
```

Comment:
It's another subtle quirk of English: this should be _5 second timeout_ (singular), not _5 seconds timeout_ (plural).